.eslintrc.js

```diff
@@ -37,6 +37,7 @@ module.exports = {
 				// Making use of typescript no-shadow instead, fixes issues with enum.
 				'no-shadow': 'off',
 				'@typescript-eslint/no-shadow': [ 'error' ],
+				'@typescript-eslint/no-empty-function': 'off',
```

Comment:
I agree to switching this rule off, its annoying

Reply:
Yes! I wasn't sure whether to commit this, but I figured I'd see what others thought.